### DISCUSSION TITLE
CF Officers: Advisory Board

### DISCUSSION THREAD

**Discussion by ebullient** [commonhaus/foundation#221] [Consensus Building]
[![🗳️ Vote progress](https://www.commonhaus.org/votes/commonhaus/foundation/221.svg)](https://github.com/commonhaus/foundation/discussions/221#discussioncomment-11775432 "DC_kwDOKRPTI84As63I")

### Proposal

I would like to nominate @ashni-mehta to help run our Advisory Board. Ashni helped set up our initial structure while at MongoDB. She has moved on from that role, and I would love to have her help with vendor outreach in 2025.

### Voting group

@commonhaus/cf-council

### Consensus mechanism

Please use emoji reactions to indicate your support:
- **👍 (:+1:)** if it looks good to you
- **👀 (:eyes:)** if you're "ok" with it (it may not be your favorite)
- **👎 (:-1:)** if you think this needs discussion first

Ask questions or raise concerns in comments.

### Vote counting instructions

Marthas

**Comment by maxandersen** [comment]
just seeing this now coming back from holidays. sorry if I'm being daft here - but what is the vote on? can't anyone help out ? or is just more informative?

**Reply by ebullient** [reply]
Informative for the rest of you, votes required from @kenfinnigan and @cealsair

[a comment by an automated account is not shown]

**Comment by ebullient** [comment]
vote::result CF Council approved. Welcome @ashni-mehta! 🎉

**Reply by JooHyukKim** [reply]
Welcome! @ashni-mehta

**Reply by ashni-mehta** [reply]
incredible news, excited to join you all!
